### PR TITLE
[multimodal][ci] adding multimodal tests

### PR DESCRIPTION
**Pull request by siddvenk**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Comment by lanking520** [comment]
should we just close or migrate this one? https://github.com/deepjavalibrary/djl-serving/pull/2157

File: tests/integration/llm/prepare.py [hunk 1]
```diff
@@ -574,6 +574,16 @@
         "option.model_id": "s3://djl-llm/llama-2-tiny/",
         "option.quantize": "awq",
         "option.tensor_parallel_degree": 4
+    },
+    "llava_v1.6-mistral": {
+        "option.model_id": "s3://djl-llm/llava-v1.6-mistral-7b-hf",
+    },
+    "paligemma-3b-mix-448": {
+        "option.model_id": "s3://djl-llm/phi-3-vision-128k-instruct"
```
Comment (lanking520) [inline tests/integration/llm/prepare.py:582]: url here is not right

Comment (siddvenk) [reply tests/integration/llm/prepare.py:582]: good catch

**Comment by siddvenk** [comment]
> should we just close or migrate this one? #2157

I think we can close that one - we don't need a lot of the configs that are there, and while the openai client is useful - I'd rather take that up separately and have a full Test suite for chat completions use-case/integ tests

**Review by lanking520 — APPROVED** [review]
